src/main/middlewares/sanitizeRequestBody.ts

```diff
@@ -0,0 +1,24 @@
+import autobind from 'autobind-decorator';
+import { NextFunction, Request, RequestHandler, Response } from 'express';
+import { flow, unescape } from 'lodash';
+import { strip } from 'node-emoji';
+import sanitizer from 'sanitizer';
+import traverse from 'traverse';
+
+@autobind
+export class SanitizeRequestBody {
+  public sanitize(req: Partial<Request>, res: Response, next: NextFunction): RequestHandler | void {
+    try {
+      const santizeValue = flow([strip, sanitizer.sanitize, unescape]);
```

Comment:
`sanitize` should take care of unsafe html tags, removing them. Given that this lib is very old and not clear what it considers as unsafe html tags, I am a bit worried that it might not be enough. In addition, next step is to unescape, so string like this `&lt;script&gt;alert('hiya')&lt;/script&gt;` come out after this as `<script>alert('hiya')</script>`. Seems like nunjucks is taking care of properly escaping that string at the display time, but are we sure that all other systems are safe from this?